### PR TITLE
new interface for VRT conversion

### PR DESCRIPTION
**Pull request by RoseRompkxm**
- remove daily conversion limit 
- adopt new contract interfaces

### CODE REVIEW

**Comment by kkirka** [comment]
Task linked: [CU-25cn8ag Display total vested amount, amount eligible for withdrawal in the UI](https://app.clickup.com/t/25cn8ag)

**Comment by kkirka** [comment]
Task linked: [CU-25cn85y Remove daily limit from frontend](https://app.clickup.com/t/25cn85y)

File: src/components/VrtConversion/Withdraw.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { Icon } from 'antd';
```
Comment (baomastr) [inline src/components/VrtConversion/Withdraw.tsx]: what do you think about deprecation of using antd and starting to use v2 components for new components in frontend?

File: src/components/VrtConversion/Withdraw.tsx [hunk 2]
```diff
@@ -37,26 +38,34 @@ export type WithdrawPropsType = {
...
-        disabled={!account}
-        onClick={handleClickWithdraw}
-      >
-        {!account ? 'Connect' : 'Withdraw'}
-      </button>
-    </ButtonWrapper>
-  </WithdrawWrapper>
-);
+      <ButtonWrapper>
+        <button
```
Comment (baomastr) [inline src/components/VrtConversion/Withdraw.tsx]: we already have v2/Button component, so you can start using it for new components

Comment (RoseRompkxm) [reply src/components/VrtConversion/Withdraw.tsx]: got it

File: src/components/VrtConversion/Withdraw.tsx (hunk 1 above)
Comment (RoseRompkxm) [reply src/components/VrtConversion/Withdraw.tsx]: make sense, I will add a new `Icon` component in v2

**Review by coreyar — APPROVED** [review]

**Pull request merged**